pfam_experiment.py

```diff
@@ -87,142 +127,171 @@ def main(_):
 	for _ in range(15001, 16001):
```

Comment:
generally you should avoid burying 'magic numbers' like this in the code. Either make it a module-level constant or a command line flag, with these values as the defaults.